locales/en/data-classes.ftl

```diff
@@ -101,6 +103,7 @@ smoking-habits = Smoking habits
 sms-messages = SMS messages
 social-connections = Social connections
 social-media-profiles = Social media profiles
+social-security-numbers = Social security numbers
```

Comment:
SSNs sounds scary. Should we add it to the priority data classes list? https://github.com/mozilla/blurts-server/blob/d99aa8133628202a15ce2471869bf5e3714ed04f/template-helpers/breach-detail.js#L44

Reply:
Definitely yes.

Reply:
We can also do it in a separate pr, I just wanted to call it out while I saw it.
But if this PR is ready and has @flodolo’s blessing, I’d rather land it quickly and not block on the priority data class stuff.

Reply:
I'd really like this to land soon. But I also need a comment on the *bios* thing

Reply:
@flodolo already done, see https://github.com/mozilla/blurts-server/pull/923/files#diff-bd84432b6e22fdc22c58a93217eeefa9

Reply:
Then yes, it should land ASAP